content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -177,6 +177,28 @@ ClusterRole with the `get` verb for the `/metrics/resources` non-resource URL.
 On Kubernetes 1.21 you must use the `--show-hidden-metrics-for-version=1.20`
 flag to expose these alpha stability metrics.
 
+### kubelet Pressure Stall Information (PSI) metrics
+
+{{< feature-state for_k8s_version="v1.33" state="alpha" >}}
+
+As an alpha feature, Kubernetes lets you configure Kubelet to collect Linux kernel [Pressure Stall Information](https://docs.kernel.org/accounting/psi.html)
+ (PSI) for CPU, memory and IO usage. The information is collected at node, pod and container level. The metrics are exposed at the `/metrics/cadvisor` endpoint with the following names:
```

Comment:
Please wrap these long lines as we do for other paragraphs.
See sembr.org for the suggested line breaking style.

```suggestion
As an alpha feature, Kubernetes lets you configure Kubelet to collect Linux kernel
[Pressure Stall Information](https://docs.kernel.org/accounting/psi.html)(PSI)
for CPU, memory and I/O usage.
The information is collected at node, pod and container level.
The metrics are exposed at the `/metrics/cadvisor` endpoint with the following names:
```